<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="stack_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QScrollArea" name="stack_scrollarea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>311</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="stack_scrollarea_contents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>449</width>
        <height>309</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="stack_label">
         <property name="font">
          <font>
           <family>.AppleSystemUIFontMonospaced</family>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="input">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>365</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFontMonospaced</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="input_symbol">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>365</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFontMonospaced</family>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="program_tabs">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>701</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFontMonospaced</family>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <widget class="QWidget" name="varw_tab">
     <attribute name="title">
      <string> VARW </string>
     </attribute>
     <widget class="QGroupBox" name="varw_groupbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QScrollArea" name="varw_scrollarea">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>321</width>
         <height>471</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="varw_scrollarea_contents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>319</width>
          <height>469</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="varw_current_vars_label">
           <property name="font">
            <font>
             <family>.AppleSystemUIFontMonospaced</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QPushButton" name="varw_done_button">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>100</y>
         <width>51</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>✓</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="varw_tools_groupbox">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>321</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="varw_value_lineedit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="varw_name_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>.AppleSystemUIFontMonospaced</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="varw_name_lineedit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="varw_value_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>.AppleSystemUIFontMonospaced</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Value:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="fnwrtr_tab">
     <attribute name="title">
      <string> FNWRTR </string>
     </attribute>
     <widget class="QGroupBox" name="fnwrtr_groupbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QScrollArea" name="fnwrtr_writer_scrollarea">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>321</width>
         <height>431</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="function_writer_scrollarea_contents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>319</width>
          <height>429</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPlainTextEdit" name="fnwrtr_textedit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QPushButton" name="fnwrtr_done_button">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>450</y>
         <width>51</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>✓</string>
       </property>
      </widget>
      <widget class="QScrollArea" name="function_writer_scrollarea_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>321</width>
         <height>471</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="function_scrollarea_contents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>319</width>
          <height>469</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="fnwrtr_current_macros_label">
           <property name="font">
            <font>
             <family>.AppleSystemUIFontMonospaced</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="matrw_tab">
     <attribute name="title">
      <string> MATRW </string>
     </attribute>
     <widget class="QGroupBox" name="matrw_groupbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="matrw_table_groupbox">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>491</width>
         <height>471</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QTableWidget" name="matrw_tablewidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>471</width>
          <height>451</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>.AppleSystemUIFontMonospaced</family>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="matrw_done_button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>51</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>✓</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QSpinBox" name="matrw_cols_spinbox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>48</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maximum">
         <number>18</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLabel" name="matrw_cols_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>.AppleSystemUIFontMonospaced</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Cols</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="matrw_rows_spinbox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>48</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maximum">
         <number>18</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLabel" name="matrw_rows_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>.AppleSystemUIFontMonospaced</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Rows</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="log_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>471</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QScrollArea" name="log_scrollarea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>131</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="log_scrollarea_contents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>449</width>
        <height>129</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="log_scrollarea_label">
         <property name="font">
          <font>
           <family>.AppleSystemUIFontMonospaced</family>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
